tracee-ebpf/tracee/argprinters.go

```diff
@@ -124,6 +141,15 @@ func (t *Tracee) parseArgs(ctx *context, args map[string]interface{}) error {
 	return nil
 }
 
+func setArgumentType(name, typeName string, argMetas *[]external.ArgMeta) {
```

Comment:
Is there any use case where we pass to the function something that is not `"string"`?

Reply:
parseArgs() is currently used for pretty-printing, so argument type is always a string